sdk/monitor/monitor-ingestion/samples-dev/earlyAborting.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates aborting additional processing early if user handle the error and decide continuing is hopeless
+ */
+
+import { DefaultAzureCredential } from "@azure/identity";
+import {
+  isAggregateUploadLogsError,
+  LogsIngestionClient,
+  UploadLogsError,
+} from "@azure/monitor-ingestion";
+
+require("dotenv").config();
+
+async function main() {
+  const logsIngestionEndpoint = process.env.LOGS_INGESTION_ENDPOINT || "logs_ingestion_endpoint";
+  const streamName = process.env.STREAM_NAME || "data_stream_name";
+  const credential = new DefaultAzureCredential();
+  const client = new LogsIngestionClient(logsIngestionEndpoint, credential);
+  let abortController = new AbortController();
+
+  const errorCallback = async function errorCallback(uploadLogsError: UploadLogsError) {
```

Comment:
Feels weird to declare the function with a name *and* assign it to a variable
```suggestion
  async function errorCallback(uploadLogsError: UploadLogsError) {
```